docs/install-binary.md

```diff
@@ -1 +1,152 @@
+
+    For production settings, Nakama should be used with CockroachDB as queries are optimised due to the way CockroachDB stores data on the disk.
+
+## Install Nakama on MacOS
+
+You’ll need to be running MacOS 10.9 (Mavericks) to run Nakama.
+
+### Homebrew
+
+Homebrew is a community-developed package manager for MacOS. If you’ve used `apt` or `yum` on Linux systems then you’ll find Homebrew to be familiar.
+
+If this is your first time using Homebrew, take a look at [their website](http://brew.sh/) for installation instructions. Once you’ve installed Homebrew, follow the instructions below.
+
+```shell fct_label="Shell"
+# run our brew recipe to download the Nakama code and build the binary
+brew install https://raw.githubusercontent.com/heroiclabs/nakama/master/install/local/nakama.rb
+```
+
+Now you have Nakama running and you’re ready to start developing.
+
+### Without Homebrew
+
+Installing the binaries directly rather than via Homebrew allows you the flexibility of placing the server in your prefered workspace. However, installing through [Homebrew](#homebrew) is the simplest and easiest – including for updates – as everything is handled using the Homebrew workflow.
+
+1. Download the latest [Nakama tarball for Mac OSX](https://github.com/heroiclabs/nakama/releases/latest).
+2. Then extract the binary:
+
+```shell fct_label="Shell"
+# replace the X.X.X with the version number you have downloaded
+tar xfz nakama-X.X.X-darwin-amd64.tar.gz
+```
+
+3. Add the directory containing the binary to your `PATH`. This makes it easy to execute Nakama commands from your terminal.
+
+```shell fct_label="Shell"
+cp -i nakama /usr/local/bin
+```
+
+4. Migrate the database schema and then start Nakama
+
+```shell fct_label="Shell"
+# migrate schema
+nakama migration up
+# start the server and output logs to the terminal
+nakama --log.stdout
+```
+
+You may come across a permissions error. If you’re happy to perform the action with root permissions then prefix the command with `sudo`.
+
+## Install Nakama on Windows
+
+Nakama is also available as a [Windows binary](https://github.com/heroiclabs/nakama/releases/latest). However, [Docker](install-docker-quickstart.md) is the recommended way to install CockroachDB and Nakama on Windows.
+
+1. [Download the binary](https://github.com/heroiclabs/nakama/releases/latest) and decompress the downloaded file.
+2. Open a Powershell terminal and navigate to the folder you downloaded Nakama into.
+3. Migrate the database schema and then start Nakama
+
+```shell fct_label="Powershell"
+# migrate schema
+nakama.exe migration up
+# start the server and output logs to the terminal
+nakama.exe --log.stdout
+```
+
+## Install Nakama on Linux
+
+Using [Docker](install-docker-quickstart.md) - This is the recommended approach. We will soon provide packages for various Linux package managers to ease the installation procedure.
```

Comment:
We could add a line here such as "If you have requests for specific package managers, please let us know so we can prioritise them"  - bit of a call to action